README.md

```diff
@@ -71,40 +74,44 @@ You will then need to generate a ID clients OAuth 2.0, download the JSON file, r
   "default_outro": "Z:/Content Creation/Processor/Assets/Outro/Outro.mp4",
   "prepend_intro": true,
   "append_outro": true,
-  "intro_outro_air_date": "2017/06/01 00:00:00"
+  "intro_outro_air_date": "2017/06/01 00:00:00",
+  "endscreen_source_video_id": "VIDEO_ID"
```

Comment:
A new field to specify the endscreen source video to copy from (was hardcoded to yours before). Obviously if you don't specify it, it will default to your value so it's backward compatible.